doc/specs/#117-Windows-Store-Support.md

```diff
@@ -0,0 +1,144 @@
+---
+author: <kevin> <larkin> <kevinlams/<kevinla@microsoft.com>
+created on: <2020-05-28>
+last updated: <2020-05-28>
+issue id: <github issue id>
+---
+# ===== DRAFT ======
```

Comment:
I think you can use a "DRAFT" PR rather than committing a document with draft in the title. If you merge this, you will have to come back and change it to remove this line.